CRM/Admin/Form/RelationshipType.php

```diff
@@ -60,7 +60,8 @@ protected function setEntityFields() {
     $this->entityFields = [
       'label_a_b' => [
         'name' => 'label_a_b',
-        'description' => ts("Label for the relationship from Contact A to Contact B. EXAMPLE: Contact A is 'Parent of' Contact B.")
+        'description' => ts("Label for the relationship from Contact A to Contact B. EXAMPLE: Contact A is 'Parent of' Contact B."),
+        'required' => TRUE,
```

Comment:
accidentally changed to non-mandatory in previous commit

Reply:
This bit should probably be in it's own PR as it's unrelated to the other changes

Reply:
thanks!

Reply:
@mattwire Ok - I could pull that part out & do the required bit first & make this depend on that - either way but there is an interdependency

Reply:
Confirmed this change is correct.  Let's leave it in the PR.

Reply:
We should add 'required' to the docblock above too `   *  - required (Is this field required or not?)`